### PR TITLE
Err dialog mobx

### PR DESCRIPTION
**Pull request by henryzbw**

### CODE REVIEW

File: src/scenes/CreateEvent/store.js [hunk 1]
```diff
@@ -305,28 +307,40 @@ export default class CreateEventStore {
   }
 
   isValidAddress = async () => {
```
Comment (dwalintukan) [inline src/scenes/CreateEvent/store.js]: needs `@action` decorator since we are calling runInAction?

File: src/scenes/Event/store.js [hunk 2]
```diff
@@ -556,128 +599,146 @@ export default class EventStore {
...
-      voteArrays.push(_.map(voteBalances.data.result[0], satoshiToDecimal));
     }
-
-    // Sum all arrays by index into one array
-    const bets = _.map(_.unzip(betArrays), _.sum);
-    const votes = _.map(_.unzip(voteArrays), _.sum);
-    return { bets, votes };
   }
 
   getEscrowAmount = async () => {
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: `@action`?

File: src/scenes/Event/store.js [hunk 3]
```diff
@@ -556,128 +599,146 @@ export default class EventStore {
   }
 
   getBetAndVoteBalances = async () => {
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: `@action`?

File: src/scenes/Event/store.js [hunk 4]
```diff
@@ -428,64 +456,79 @@ export default class EventStore {
...
+      });
+    } catch (error) {
+      runInAction(() => {
+        this.app.ui.setError(error.message, `${networkRoutes.graphql.http}/createFinalizeResultTx`);
+      });
+    }
     Tracking.track('oracleDetail-finalize');
   }
 
   withdraw = async (senderAddress, type) => {
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: `@action`?

File: src/scenes/Event/store.js [hunk 5]
```diff
@@ -428,64 +456,79 @@ export default class EventStore {
...
+    } catch (error) {
+      runInAction(() => {
+        this.app.ui.setError(error.message, `${networkRoutes.graphql.http}/createVoteTx`);
+      });
+    }
 
     Tracking.track('oracleDetail-vote');
   }
 
   finalize = async () => {
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: `@action`?

File: src/scenes/Event/store.js [hunk 6]
```diff
@@ -556,128 +599,146 @@ export default class EventStore {
...
+      const eventEscrowAmount = satoshiToDecimal(escrowRes.data.result[0]);
     return eventEscrowAmount;
+    } catch (error) {
+      runInAction(() => {
+        this.app.ui.setError(error.message, networkRoutes.api.eventEscrowAmount);
+      });
+    }
   }
 
   getWithdrawableAddresses = async () => {
```
Comment (dwalintukan) [inline src/scenes/Event/store.js]: `@action`?

File: src/scenes/CreateEvent/store.js (hunk 1 above)
Comment (henryzbw) [reply src/scenes/CreateEvent/store.js]: I think the runInAction only means the code it wraps will run as an action,  but the isValidAddress doesn't change any observable, so we can leave it as a normal function.
same as next comments

**Pull request merged**